client/src/components/SignIn.jsx

```diff
@@ -67,11 +83,18 @@ class SignIn extends React.Component {
 
       } else {
         // Failed Login 
+        console.log(data)
```

Comment:
Can we either remove this or something about it being a failed login? Just something like console.log('Failed Login:', data). Probably shouldn't print out the password that didn't work.

Reply:
Yep I accidentally left that behind. It has been removed.